modules/op-constructing-pipelines-using-pipeline-builder.adoc

```diff
@@ -0,0 +1,43 @@
+// This module is included in the following assembly:
+//
+// *openshift_pipelines/working-with-pipelines-using-the-developer-perspective.adoc
+
+[id="op-constructing-pipelines-using-pipeline-builder_{context}"]
+= Constructing Pipelines using the Pipeline Builder
+
+You can use the Pipeline Builder to:
+
+* Construct a Pipeline flow using existing Tasks and CLusterTasks. When you install the OpenShift Pipelines Operator, it adds reusable Pipeline ClusterTasks to your cluster.
+* Specify the type of resources required for the Pipeline execution, and if required, add additional parameters to the Pipeline.
+* Reference these Pipeline resources in each of the Tasks in the Pipeline as input and output resources.
+* The parameters for a Task are prepopulated based on the specifications of the Task. If required, reference the additional parameters added to the Pipeline in the Task.
+
+.Procedure
+
+. In the *Add* view of the *Developer* perspective, click the *Pipeline* tile to see the *Pipeline Builder* page.
+. Enter a unique name for the Pipeline.
+. Select a Task from the *Select task* list to add a Task to the Pipeline. For this example, select a *s2i-nodejs* Task.
+.. To add sequential Tasks to the Pipeline, click the plus icon to the right or left of the Task, and from the *Select task* list, select the Task you want to add to the Pipeline. For this example, use the plus icon to the right of the *s2i-nodejs* Task to add an *openshift-client* Task.
+.. To add a parallel Task to the existing Task, click the plus icon displayed below the Task, and from the *Select Task* list, select the parallel Task you want to add to the Pipeline.
++
+image::op-pipeline-builder.png[Pipeline Builder]
++
+. Click *Add Resources* to specify the name and type of resources that the Pipeline will need to use during execution. These resources are then used by the Tasks in the Pipeline as inputs and outputs.
+For this example:
+.. Add an input resource. In the *Name* field, enter `Source`, and from the *Resource Type* drop-down list, select *Git*.
```

Comment:
As a convention, we mention the location of action first and then the action so that the user knows which section he needs to focus on to complete the action.